cli/command/manifest/fetch.go

```diff
@@ -0,0 +1,326 @@
+package manifest
```

Comment:
So let's design it. Putting everything in the same package is not fixing that problem, it's just making it harder to fix the problem later because all the code will be tangled.

There is a clear responsibility for the packages in `cli/command/` and this code does not fit.